tests/unit/test_dimension.py

```diff
@@ -1217,6 +1222,26 @@ def but_it_raises_when_insertion_id_not_present(self, _order_dict_prop_):
             _OrderSpec(None, None).insertion_id
         assert str(e.value) == "'insertion_id'"
 
+    def it_knows_the_marginal_specified_as_the_sort_basis(self, _order_dict_prop_):
+        _order_dict_prop_.return_value = {"marginal": "scale_mean"}
+        assert _OrderSpec(None, None).marginal == MARGINAL.SCALE_MEAN
+
+    def but_it_raises_when_marginal_field_is_not_present(self, _order_dict_prop_):
+        _order_dict_prop_.return_value = {}
+        with pytest.raises(KeyError) as e:
+            _OrderSpec(None, None).marginal
+        assert str(e.value) == "'marginal'"
+
+    def and_it_raises_when_the_marginal_keyword_is_not_recognized(
+        self, _order_dict_prop_
+    ):
+        _order_dict_prop_.return_value = {"marginal": "foobar"}
+        with pytest.raises(ValueError) as e:
+            _OrderSpec(None, None).marginal
+        # --- `.endswith()` to accommodate a Python 2.7 vs. 3.0 error message difference
+        # --- in Enum module
+        assert str(e.value).endswith(" is not a valid MARGINAL")
```

Comment:
Probably this one too because once migration is done all the enum keywords will be defined. The exception should and will still arise if there's some development-time error, but I don't think we need to verify that it does. We're not the ones raising the exception. Come to think of it, I think that's the rule-of-thumb; only test exceptions that you yourself raise in the code.